book/installation.md

```diff
@@ -1,6 +1,46 @@
 ## Installation
 
-(still WIP, please contribute!):
+To test Reth, you will need to have Geth and Lighthouse installed. 
+
+#### Mac
+```bash
+brew install geth lighthouse
+```
+
+#### Ubuntu
+1. First, update the package list:
+```bash
+sudo apt-get update
+```
+
+
+2. Then, install Geth:
+```bash
+sudo apt-get install geth
+```
+
+
+To install Lighthouse, you will need to follow the instructions on the Lighthouse GitHub page (https://github.com/sigp/lighthouse). This typically involves cloning the repository and building from source.
+
+Here is the steps to install Lighthouse on Ubuntu:
+
+1. Install the dependencies:
+```bash
+sudo apt-get install build-essential pkg-config libssl-dev libcurl4-openssl-dev libgmp-dev libboost-all-dev libsecp256k1-dev
+```
+
+2. Clone the repository:
+```bash
+git clone https://github.com/sigp/lighthouse.git
+```
```

Comment:
we definitely should link to install docs here.

I guess these instructions are unlikely to change much but maybe we remove them here and only add links?

Reply:
Agree only links